### PR TITLE
Fixed expand issue

### PR DESCRIPTION
**Pull request by gnehapk**
Bugzilla: 1596655
tendrl-bug-id: https://github.com/Tendrl/ui/issues/1025

Signed-off-by: Neha Gupta <gnehapk@gmail.com>

### CODE REVIEW

File: src/modules/hosts/host-list/host-list.js [hunk 1]
```diff
@@ -15,7 +15,7 @@
         });
 
     /*@ngInject*/
-    function hostController($scope, $filter, $rootScope, $state, $interval, $uibModal, utils, config, nodeStore, clusterStore) {
+    function hostController($scope, $filter, $rootScope, $state, $interval, $uibModal, utils, config, nodeStore, clusterStore, Notifications) {
```
Comment (cloudbehl) [inline src/modules/hosts/host-list/host-list.js]: why we injected `Notifications`? I don't see any use of it in this PR?

Comment (a2batic) [reply src/modules/hosts/host-list/host-list.js]: @cloudbehl, `Notifications` is used in other places on the view, and might have left out earlier to be added. (Line 92, 94)

**Review by a2batic — DISMISSED** [review]

File: src/modules/hosts/host-list/host-list.js (hunk 1 above)
Comment (cloudbehl) [reply src/modules/hosts/host-list/host-list.js]: @a2batic ok, makes sense! But let's follow the process.

Let the program take call on this and if they feel it's critical then we will fix it then will do it. what you say?

Can you create a another Issue for this bug?

Comment (cybernth) [reply src/modules/hosts/host-list/host-list.js]: Let's not take in this change now. revert back the notification related change, create a git issue for the same. I am not very keen to take this in for 3.4.0. Let's fix this in the next update.

**Comment by cloudbehl** [comment]
@shtripat can you review the logic once? I am not sure if this will work if the profiling is enabled in between.

If required @gnehapk, you and me can get in a bluejeans call.

**Review by shtripat — COMMENTED** [review]
Is this the same condition we used earlier in cluster list menu option for enabling the expand button?

**Review by a2batic — APPROVED** [review]

**Review by cloudbehl — APPROVED** [review]
Non-managed is closer in meaning to not managed than is unmanaged. E.g. unmanaged can be used with the specific meaning no longer managed, but non-managed cannot.

We need to write a more meaningful variable name.


The code Looks good to me!

**Review by cybernth — APPROVED** [review]

**Pull request merged**